config/locales/en-GB.yml

```diff
@@ -12899,7 +12899,7 @@ en-GB:
       default_assignment_settings: "**Default Assignment Settings**"
       default_comments_settings: "*Text entered here will be displayed to students when submitting an assignment*"
       labels: 
-        account_id: "VeriCite Account ID"
+        account_id: "VeriCite Consumer Key"
```

Comment:
These translation files are generated elsewhere and don't need to receive hand updates.